api-samples/declarativeNetRequest/no-cookies-manager/popup.js

```diff
@@ -0,0 +1,24 @@
+document.getElementById('openManagerTab').addEventListener('click', () => {
+  chrome.tabs.create({ url: 'manager.html' });
+});
+
+const matchedRulesList = document.getElementById('matchedRulesList');
+
+async function init() {
+  const tabId = (await chrome.tabs.query({ active: true }))[0].id;
+  const result = await chrome.declarativeNetRequest.getMatchedRules({ tabId });
+  if (result.rulesMatchedInfo.length === 0) {
+    document.querySelector('.tips').style.display = 'block';
+    return;
+  }
+  document.querySelector('.matched-rules').style.display = 'block';
```

Comment:
We should probably have some logic to hide the tips if everything works?

Reply:
![image](https://github.com/GoogleChrome/chrome-extensions-samples/assets/18638914/546670b3-824a-40d0-85e9-32bcf02802f2)
Actually, `matched-rules` is the container that holds the results. I think it might be better to name it `matched-rules-wrapper`?

Reply:
Sorry, my comment was probably on an unhelpful line. I was just wondering if we should add this?

```
document.querySelector('.tips').style.display = 'none';
```

Otherwise I think the tips will never go away?

Reply:
Oh, I understand what you mean. 
But I think this tips itself should not be removed. When the user opens the popup, if the extension does not find a matching rule, this element should always be present during the time the popup is open.

Reply:
Ah, ok. That makes sense!